index/scorch/segment/zap/docvalues.go

```diff
@@ -135,6 +137,40 @@ func (di *docValueReader) loadDvChunk(chunkNumber uint64, s *SegmentBase) error
 	return nil
 }
 
+func (di *docValueReader) iterateDocTerms(s *SegmentBase, visitor docNumTermsVisitor) error {
+	for i := uint64(0); i < di.numChunks; i++ {
+		err := di.loadDvChunk(i, s)
+		if err != nil {
+			return err
+		}
+
+		var uncompressed []byte
+		// use the uncompressed copy if available
+		if len(di.uncompressed) > 0 {
```

Comment:
after calling loadDvChunk(), looks like len(di.uncompressed) will always be 0?  so, don't need the if-check here and the else-clause codepath always runs?